nixos/modules/services/networking/unbound.nix

```diff
@@ -86,6 +91,28 @@ in
         description = "Use and update root trust anchor for DNSSEC validation.";
       };
 
+      localControlSocketPath = mkOption {
+        default = null;
+        # FIXME: What is the proper type here so users can specify strings,
```

Comment:
I'm fine with keeping the current `types.str` alone. `types.path` can be relative to the $pwd at evaluation time. While it's what you want to point to a file living in your source tree, I always find awkward using this type to point to a runtime path.

Reply:
Same but often my taste doesn't seem to follow what some in the community are pushing for.